packages/sitecore-jss-nextjs/src/services/graphql-sitemap-service.ts

```diff
@@ -204,7 +209,9 @@ export class GraphQLSitemapService {
    */
   protected async fetchSitemap(
     languages: string[],
-    formatStaticPath: (path: string[], language: string) => StaticPath
+    formatStaticPath: (path: string[], language: string) => StaticPath,
+    includedPaths?: string[],
+    excludedPaths?: string[]
```

Comment:
Why were these parms added? Doesn't look like these are used (by the 2 public functions that call it). If the intent was to add these to the public fetch functions, then we should exclude from the service options (would be confusing to have in both places).

Reply:
done